.github/workflows/build.yml

```diff
@@ -10,7 +10,12 @@ jobs:
     steps:
 
     - uses: actions/checkout@v1
-      
+
+    - name: Set up .NET 5.0
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 5.0.100
```

Comment:
This should probably be updated to use the latest SDK, `5.0.200` as of today, or `5.0.x` to use the latest automatically.